CMakeLists.txt

```diff
@@ -72,9 +72,7 @@ option (USE_OPENGL "Include OpenGL support" ON)
 option (USE_QT "Include Qt support" ON)
 option (FORCE_OPENGL_1 "Force iv to use OpenGL's fixed pipeline")
 option (USE_PYTHON "Build the Python bindings" ON)
-option (USE_PYTHON3 "Build the Python3 bindings" OFF)
-set (PYTHON_VERSION 2.6)
-set (PYTHON3_VERSION 3.2)
+set (PYTHON_VERSION "2.7" CACHE STRING "Target version of python to find")
```

Comment:
Is min version still 2.6 or now 2.7? But can be changed with a configure flag, anyways.

Reply:
I think probably 2.7 is the proper default minimum, since that has been the case for VFXPlatform since as early as 2014 (this year, it's specifically 2.7.5 minimum). Also note that VFXPlatform is planning a transition to Python 3 in CY 2019.

Reply:
I'll leave the minimum as 2.7 then.